recipes/bioconductor-pharmacogx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.10.3" %}
+{% set name = "PharmacoGx" %}
+{% set bioc = "3.7" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: '{{ version }}'
+source:
+  fn: '{{ name }}_{{ version }}.tar.gz'
+  url:
+    - 'http://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
+    - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
+    - 'https://depot.galaxyproject.org/software/bioconductor-{{ name|lower }}/bioconductor-{{ name|lower }}_{{ version }}_src_all.tar.gz'
+  sha256: 7dfb2e1b4fbe3e0842e46f4e00b704dae0dc0cf5decf37c9dd652ea3e634c742
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  host:
+    - bioconductor-biobase
+    - bioconductor-piano
+    - r-base
+    - r-catools
+    - r-downloader
+    - r-lsa
+    - r-magicaxis
+    - r-rcolorbrewer
+    - r-reshape2
+  run:
+    - bioconductor-biobase
+    - bioconductor-piano
+    - r-base
+    - r-catools
+    - r-downloader
+    - r-lsa
+    - r-magicaxis
+    - r-rcolorbrewer
+    - r-reshape2
+test:
+  commands:
+    - '$R -e "library(''{{ name }}'')"'
+about:
+  home: 'http://bioconductor.org/packages/{{ bioc }}/bioc/html/{{ name }}.html'
+  license: Artistic-2.0
+  summary: 'Contains a set of functions to perform large-scale analysis of pharmacogenomic data.'
+
+extra:
+  skip-lints:
+    - should_not_use_fn
```

Comment:
`fn` is no longer required with conda3

Reply:
Thanks for the catch @corburn - the recipe is fixed now.